src/behavioural-events/events.ts

```diff
@@ -1,49 +1,79 @@
+import { camelToUnderscore } from "../helpers/camel-to-underscore";
 import { v4 as uuidv4 } from "uuid";
 
-type EventType = "web_billing_sdk_initialized";
+enum EventType {
+  SDK_INITIALIZED = "web_billing_sdk_initialized",
+  CHECKOUT_SESSION_START = "web_billing_checkout_session_start",
```

Comment:
I just realized that in other enums we use `PascalCase`. Should do the same here?

Reply:
Ah they where first constants, maybe that is why. I will review. Thanks